src/Components/MovieList/index.tsx

```diff
@@ -1,18 +1,83 @@
-import React from 'react'
+import React, { useReducer } from 'react'
 import Movie from 'Components/Movie'
-import { MoviesResponseModel } from './models'
+import { MovieListLocalState, MovieListReducerAction, MoviesResponseModel } from './models'
 import MoviesMockData from './movies.json'
-import './styles.scss'
 import { MoviesHeader } from './Header'
+import { Modal } from 'Components'
+import MovieForm from 'Components/MovieForm'
+import { MovieFormAction } from 'Components/MovieForm/models'
+import './styles.scss'
+import { MovieModel } from 'Components/Movie/models'
+
+
```

Comment:
you can remove these empty lines

Reply:
Freankly speaking I am confusing, I don't see this emply lines in my code, but GitHub shows previous version of file before formated by prettier